lib/matplotlib/backends/backend_qt5.py

```diff
@@ -489,14 +491,23 @@ def __init__(self, canvas, num):
 
         self.window._destroying = False
 
-        # add text label to status bar
-        self.statusbar_label = QtWidgets.QLabel()
-        self.window.statusBar().addWidget(self.statusbar_label)
-
+        self.toolmanager = self._get_toolmanager()
         self.toolbar = self._get_toolbar(self.canvas, self.window)
+        self.statusbar = None
+
+        if self.toolmanager:
+            backend_tools.add_tools_to_manager(self.toolmanager)
+            if self.toolbar:
+                backend_tools.add_tools_to_container(self.toolbar)
+                self.statusbar = StatusbarQt(self.window, self.toolmanager)
+
         if self.toolbar is not None:
             self.window.addToolBar(self.toolbar)
-            self.toolbar.message.connect(self.statusbar_label.setText)
+            if not self.toolmanager:
+                # add text label to status bar
+                statusbar_label = QtWidgets.QLabel()
```

Comment:
This code used to happen regardless of `self.toolbar`.  Is it OK that it only happens if its none?

Reply:
toolmanager and the old navigation toolbar are different. So that widget is initialized only if there is no toolmanager

Reply:
Yes, but can `self.toolbar` be `None`?  Because even if it was, the old lines 492-494 were called.  Now they aren't.

Reply:
I really don't know if that was a mistake before.
The reasoning behind is: If you don't want a toolbar, you don't want a statusbar, we don't have separate controls but maybe we should

Reply:
Isn't the status bar the thing at the top that says "Figure 1"?  I think you want that regardless of whether there is a toolbar...  Or is it something else?